packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -100,9 +106,11 @@
                 >{{ country.name }}</option
               >
             </select>
-            <span class="error-color error-msg" v-if="form.country.errorList.length">{{
-              validationService.errorDictionary.emptyField
-            }}</span>
+            <span
+              class="error-color error-msg"
+              v-if="form.country.errorList.length"
+              >{{ validationService.errorDictionary.emptyField }}</span
+            >
```

Comment:
Is this prettier doing this?

Reply:
Odd, this not notice. I believe it is. Not sure why, must me the maximum line length of 80 chars but It should definitely not rearrange it as such.

Reply:
There's definitely some head-scratchers that prettier outputs. This might be one of those 🤷